src/Microsoft.ML/Models/OnnxConverter.cs

```diff
@@ -10,9 +10,50 @@ namespace Microsoft.ML.Models
     public sealed partial class OnnxConverter
     {
         /// <summary>
-        /// Converts the model to ONNX format.
+        ///
+        /// This API converts the model to ONNX format by inspecting the transform pipeline 
+        /// from the end, checking for components that know how to save themselves as ONNX. 
+        /// The first data view in the transform pipeline that does not know how to save itself 
+        /// as ONNX, is considered the "input" to the ONNX pipeline. (Ideally this would be the 
+        /// original loader itself, but this may not be possible if the user used unsavable 
+        /// transforms in defining the pipe.) All the columns in the source that are a type the 
+        /// ONNX knows what to deal with will be tracked. Intermediate transformations of the 
+        /// data appearing as new columns will appear in the output block of the ONNX, with names 
+        /// derived from the corresponding column names. The ONNX json will be serialized to a 
+        /// path defined through the Json option.
+        ///
+        /// This API supports the following arguments:
+        /// <see cref="Onnx"/> indicates the file to write the ONNX protocol buffer file to.
+        /// <see cref="Json"/> indicates the file to write the JSON representation of the ONNX model.
```

Comment:
>indicates the file to write the JSON representation of the ONNX model. [](start = 31, length = 70)

Is that optional?

Reply:
Yes.

---
In reply to: [198256809](https://github.com/dotnet/machinelearning/pull/419#discussion_r198256809) [](ancestors = 198256809)